src/core/util.jl

```diff
@@ -61,25 +62,21 @@ end
 
 auto_tune_chunk_size!(mf::Function, rep_num=10) = begin
   dim = length(mf().vals)
+  chunk_size = 8
```

Comment:
Perhaps we should remove this function since it's no longer used after recent AD refactoring?